pkg/source/reconciler/mtsource/controller.go

```diff
@@ -58,8 +79,8 @@ func NewController(
 ) *controller.Impl {
 	logger := logging.FromContext(ctx)
 
-	env := &envConfig{}
-	if err := envconfig.Process("", env); err != nil {
+	env, err := GetConfig()
+	if err != nil {
```

Comment:
not sure I understand why changing here. Instead of a global var I suggest to pass the value as a function parameter.

Reply:
which function? No function is called in mtsource Kafkasource.go belongs to this file.